src/HomeNet/HomeNet.conf

```diff
@@ -102,15 +102,39 @@ client_customer_interface_port = 16988
 #
 # To generate PFX certificate, you can use OpenSSL:
 #
-#   openssl req -x509 -newkey rsa:4096 -keyout HomeNet.key -out HomeNet.cer -days 36500
+#   openssl req -x509 -newkey rsa:4096 -keyout HomeNet.key -out HomeNet.cer -days 365000

```

Comment:
This number reminds me of the song "Desert Rose" from Sting. So you think the certificate needs to be valid for a 1000 years? The current rule of thumb is that you should replace GPG or PKI certificates every few years.